ChangeLog.md

```diff
@@ -1,5 +1,171 @@
 # Changelog for haskell-language-server
 
+## 0.9.0
+
+This release includes lot of refactorings and bug fixes over existing features, hlint and eval plugins among others.
+The list of contributors keeps growing healthy, many thanks to you all!
+
+And remember, we have a new brand logo, courtesy of @Ailrun :slightly_smiling_face:
+
+![haskell-language-server](https://github.com/haskell/haskell-language-server/raw/master/docs/logos/logo-256.png)
+
+### Pull requests merged for 0.9.0
+
+- Tighten dependency on apply-refact
+([#1268)](https://github.com/haskell/haskell-language-server/pull/1268) by @hololeap
```

Comment:
Out of curiosity, is there a reason for this style? I mean, the style in which including `)` as a part of the link but not `(`.

Reply:
Not sure, GenChangelo.hs script generates them that way, maybe it is unintentional? //cc @bubba